src/main/java/seedu/address/logic/commands/InternshipDeleteTaskCommand.java

```diff
@@ -48,13 +48,14 @@ public CommandResult execute(InternshipModel model) throws CommandException {
         requireNonNull(model);
         List<Internship> lastShownList = model.getFilteredInternshipList();
 
-        if (internshipIndex.getOneBased() > lastShownList.size()) {
+        if (internshipIndex.getOneBased() > lastShownList.size() || internshipIndex.getOneBased() <= 0) {
```

Comment:
unnecessary, won't be parsed here -> negative numbers are rejected when parsed to index, so this should alr be covered in parser